var/spack/repos/builtin/packages/warpx/package.py

```diff
@@ -20,7 +20,7 @@ class Warpx(MakefilePackage):
 
     maintainers = ['ax3l', 'dpgrote', 'MaxThevenet', 'RemiLehe']
 
-    version('master', tag='master')
+    version('development', tag='development')
```

Comment:
```suggestion
    version('develop', tag='development')
```
This will allow the version to sort correctly. For example, if someone says `depends_on('warpx@2:')`, `develop` will satisfy this, but `development` will not.

Reply:
Thanks for catching this!